### PR TITLE
update PLASTIK token address

### PR DESCRIPTION
**Pull request by garciapd**
the token has been migrated

### CODE REVIEW

**Comment by garciapd** [comment]
hi, can someone check why we are having errors on the merge request please?

File: src/data/mainnet/tokens-info.json [hunk 1]
```diff
@@ -416,7 +416,7 @@
     "symbol": "ETHIX"
   },
   {
-    "address": "0xf4361a8fbcf6b6a2402f1a1d6c4b945e59a29934",
+    "address": "0x27cd006548dF7C8c8e9fdc4A67fa05C2E3CA5CF9",
```
Comment (jeanregisser) [inline src/data/mainnet/tokens-info.json:419]: The address needs to be lowercased for the test to pass.

**Review by jeanregisser — COMMENTED** [review]
Thanks,

What is happening for existing tokens? Do we need to keep the old address under a different name (Plastik old?) to avoid confusion for existing holders?

File: src/data/mainnet/tokens-info.json (hunk 1 above)
Comment (garciapd) [reply src/data/mainnet/tokens-info.json:419]: hi, we have done a migration, the old token is going to be deprecated and paused (noobdy will transact), and we have done already a balance migration to the new one

Comment (jeanregisser) [reply src/data/mainnet/tokens-info.json:419]: Any issue updating the address to lowercase @garciapd? I can't merge the PR without it.

Comment (garciapd) [reply src/data/mainnet/tokens-info.json]: done in a new commit :)

**Review by jeanregisser — APPROVED** [review]

**Pull request merged**